src/main/kotlin/com/criptext/mail/scenes/emaildetail/ui/holders/FullEmailHolder.kt

```diff
@@ -210,6 +213,29 @@ class FullEmailHolder(view: View) : ParentEmailHolder(view) {
         setToText(fullEmail)
         setDraftIcon(fullEmail)
         setIcons(fullEmail.email.delivered)
+
+        val storage = KeyValueStorage.SharedPrefs(context)
```

Comment:
There is no need for this here, the controller already has a reference of the storage, keep the scenes and holders free of that logic, if you need something to be done in the shared prefs storage let the controller know so it can handle what you want.